pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -445,6 +445,16 @@
     "vendorSha256": "sha256-l2OviwplP/Sg2ShaEA88pMwVTkREnLkFAzterjr2kvU=",
     "version": "4.19.0"
   },
+  "googleworkspace": {
+    "owner": "hashicorp",
+    "provider-source-address": "registry.terraform.io/hashicorp/googleworkspace",
+    "proxyVendor": true,
```

Comment:
Doesn't seem to be needed?

```suggestion
```

Reply:
Yeah sorry, copy-pasted that without noticing ><